### PR TITLE
Added option DB_NAMES_EXCLUDE to prevent specific databases from beeing dumped

### PR DESCRIPTION
**Pull request by alexanderhofstaetter**
See #138 

With the new environment variable `DB_NAMES_EXCLUDE` one can specifiy a space seperated list of database names to exclude from mysqldump.

New default: By default the databases`information_schema`, `mysql`, `sys`and `performance_schema` are excluded to prevent errors when dumping all databases.

This only applies to the dump-by-schema mode (`DB_DUMP_BY_SCHEMA=true`) otherwise `mysqldump --all-databases` is used which excludes theese database by default

> mysqldump does not dump the performance_schema or sys schema by default. To dump any of these, name them explicitly on the command line. You can also name them with the --databases option. For performance_schema, also use the --skip-lock-tables option.

> In MySQL 8.0, the mysql schema is considered a system schema that cannot be dropped by end users.

Taken from: https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,11 +34,13 @@ The following are the environment variables for a backup:
 
 __You should consider the [use of `--env-file=`](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables-e-env-env-file), [docker secrets](https://docs.docker.com/engine/swarm/secrets/) to keep your secrets out of your shell history__
 
+* `TZ`: set the timezone used for backup timestamps, for example `TZ=Europe/Vienna`, defaults to `UTC`
 * `DB_SERVER`: hostname to connect to database. Required.
 * `DB_PORT`: port to use to connect to database. Optional, defaults to `3306`
 * `DB_USER`: username for the database
 * `DB_PASS`: password for the database
-* `DB_NAMES`: names of databases to dump; defaults to all databases in the database server
+* `DB_NAMES`: names of databases to dump (seperated by space); defaults to all databases in the database server
```
Comment (deitch) [inline README.md:42]: "separated"

File: README.md [hunk 2]
```diff
@@ -34,11 +34,13 @@ The following are the environment variables for a backup:
...
+* `DB_NAMES_EXCLUDE`: names of databases (seperated by space) to exclude from the dump; `information_schema`. `performance_schema`, `sys` and `mysql` are excluded by default. This only applies if `DB_DUMP_BY_SCHEMA` is set to `true`. For example, if you set `DB_NAMES_EXCLUDE=database1 db2` and `DB_DUMP_BY_SCHEMA=true` then theese two databases will not be dumped by mysqldump.
```
Comment (deitch) [inline README.md:43]: * "separated" (replaces "seperated")
* "these" (replaces "theese")
* `DB_NAMES_EXCLUDE="database1 db2"` (add the quotes please)

File: README.md [hunk 3]
```diff
@@ -61,7 +63,7 @@ __You should consider the [use of `--env-file=`](https://docs.docker.com/engine/
 * `COMPRESSION`: Compression to use. Supported are: `gzip` (default), `bzip2`
 * `DB_DUMP_BY_SCHEMA`: Whether to use separate files per schema in the compressed file (`true`), or a single dump file (`false`). Defaults to `false`.
 * `DB_DUMP_KEEP_PERMISSIONS`: Whether to keep permissions for a file target. By default, `mysql-backup` copies the backup compressed file to the target with `cp -a`. In certain filesystems with certain permissions, this may cause errors. You can disable the `-a` flag by setting `DB_DUMP_KEEP_PERMISSIONS=false`. Defaults to `true`.
-* `MYSQLDUMP_OPTS`: A string of options to pass to `mysqldump`, e.g. `MYSQLDUMP_OPTS="--opt abc --param def --max_allowed_packet=123455678"` will run `mysqldump --opt abc --param def --max_allowed_packet=123455678`
+* `MYSQLDUMP_OPTS`: A string of options to pass to `mysqldump`, e.g. `MYSQLDUMP_OPTS=--opt abc --param def --max_allowed_packet=123455678` will run `mysqldump --opt abc --param def --max_allowed_packet=123455678`
```
Comment (deitch) [inline README.md:66]: Why this change? This is a single opt, and should be enclosed in double-quotes

**Comment by deitch** [comment]
Hi @alexanderhofstaetter this has been open with "request for changes" for a while. Do you want to fix them up, rebase it, and we can get it in?

**Comment by evilchis94** [comment]
Hi there, this feature will be awesome, any plan on closing this? Thanks!

**Comment by deitch** [comment]
@evilchis94 it looks like @alexanderhofstaetter abandoned it. I am going to close the PR, as it is open for a year already. If you want to open a fresh one to get it in, would be open to it.
